### ISSUE TITLE
Long bug in cmdline

### ISSUE THREAD

**Issue by ghost** [python/cpython#100974]
# Bug report
Reproduce by running a python command during Python installation. This error persists after a clean installation of Python (repaired, then uninstalled, then reinstalled)

```
C:\Users\rico>py -v
import _frozen_importlib # frozen
import _imp # builtin
import '_thread' # <class '_frozen_importlib.BuiltinImporter'>
import '_warnings' # <class '_frozen_importlib.BuiltinImporter'>
import '_weakref' # <class '_frozen_importlib.BuiltinImporter'>
import 'winreg' # <class '_frozen_importlib.BuiltinImporter'>
import '_io' # <class '_frozen_importlib.BuiltinImporter'>
import 'marshal' # <class '_frozen_importlib.BuiltinImporter'>
import 'nt' # <class '_frozen_importlib.BuiltinImporter'>
import '_frozen_importlib_external' # <class '_frozen_importlib.FrozenImporter'>
# installing zipimport hook
import 'time' # <class '_frozen_importlib.BuiltinImporter'>
import 'zipimport' # <class '_frozen_importlib.FrozenImporter'>
# installed zipimport hook
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\__pycache__\__init__.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\__init__.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\encodings\\__pycache__\\__init__.cpython-311.pyc'
import '_codecs' # <class '_frozen_importlib.BuiltinImporter'>
import 'codecs' # <class '_frozen_importlib.FrozenImporter'>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\__pycache__\aliases.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\aliases.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\encodings\\__pycache__\\aliases.cpython-311.pyc'
import 'encodings.aliases' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE031A50>
import 'encodings' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE027390>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\__pycache__\utf_8.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\utf_8.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\encodings\\__pycache__\\utf_8.cpython-311.pyc'
import 'encodings.utf_8' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE0516D0>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\__pycache__\cp1252.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\encodings\cp1252.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\encodings\\__pycache__\\cp1252.cpython-311.pyc'
import 'encodings.cp1252' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE051CD0>
import '_signal' # <class '_frozen_importlib.BuiltinImporter'>
import '_abc' # <class '_frozen_importlib.BuiltinImporter'>
import 'abc' # <class '_frozen_importlib.FrozenImporter'>
import 'io' # <class '_frozen_importlib.FrozenImporter'>
import '_stat' # <class '_frozen_importlib.BuiltinImporter'>
import 'stat' # <class '_frozen_importlib.FrozenImporter'>
import '_collections_abc' # <class '_frozen_importlib.FrozenImporter'>
import 'genericpath' # <class '_frozen_importlib.FrozenImporter'>
import '_winapi' # <class '_frozen_importlib.BuiltinImporter'>
import 'ntpath' # <class '_frozen_importlib.FrozenImporter'>
import 'os' # <class '_frozen_importlib.FrozenImporter'>
import '_sitebuiltins' # <class '_frozen_importlib.FrozenImporter'>
Processing user site-packages
Processing global site-packages
Adding directory: 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311'
Adding directory: 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages'
Processing .pth file: 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\distutils-precedence.pth'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\site-packages\_distutils_hack\__pycache__\__init__.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\site-packages\_distutils_hack\__init__.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_distutils_hack\\__pycache__\\__init__.cpython-311.pyc'
import '_distutils_hack' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE1CC5D0>
import 'site' # <class '_frozen_importlib.FrozenImporter'>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\rlcompleter.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\rlcompleter.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\rlcompleter.cpython-311.pyc'
import 'atexit' # <class '_frozen_importlib.BuiltinImporter'>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\inspect.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\inspect.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\inspect.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\ast.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\ast.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\ast.cpython-311.pyc'
import '_ast' # <class '_frozen_importlib.BuiltinImporter'>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\contextlib.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\contextlib.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\contextlib.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\collections\__pycache__\__init__.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\collections\__init__.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\collections\\__pycache__\\__init__.cpython-311.pyc'
import 'itertools' # <class '_frozen_importlib.BuiltinImporter'>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\keyword.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\keyword.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\keyword.cpython-311.pyc'
import 'keyword' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE25E810>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\operator.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\operator.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\operator.cpython-311.pyc'
import '_operator' # <class '_frozen_importlib.BuiltinImporter'>
import 'operator' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE25ED50>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\reprlib.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\reprlib.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\reprlib.cpython-311.pyc'
import 'reprlib' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE25F990>
import '_collections' # <class '_frozen_importlib.BuiltinImporter'>
import 'collections' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE236210>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\functools.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\functools.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\functools.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\types.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\types.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\types.cpython-311.pyc'
import 'types' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE283210>
import '_functools' # <class '_frozen_importlib.BuiltinImporter'>
import 'functools' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE2450D0>
import 'contextlib' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE235650>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\enum.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\enum.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\enum.cpython-311.pyc'
import 'enum' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE237510>
import 'ast' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE1F4B90>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\dis.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\dis.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\dis.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\opcode.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\opcode.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\opcode.cpython-311.pyc'
import '_opcode' # <class '_frozen_importlib.BuiltinImporter'>
import 'opcode' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4D1850>
import 'dis' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE1F72D0>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\collections\__pycache__\abc.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\collections\abc.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\collections\\__pycache__\\abc.cpython-311.pyc'
import 'collections.abc' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4C2B10>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\importlib\__pycache__\__init__.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\importlib\__init__.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\importlib\\__pycache__\\__init__.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\warnings.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\warnings.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\warnings.cpython-311.pyc'
import 'warnings' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4D9310>
import 'importlib' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4D9F90>
import 'importlib.machinery' # <class '_frozen_importlib.FrozenImporter'>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\linecache.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\linecache.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\linecache.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\tokenize.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\tokenize.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\tokenize.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\__pycache__\__init__.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\__init__.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\re\\__pycache__\\__init__.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\__pycache__\_compiler.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\_compiler.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\re\\__pycache__\\_compiler.cpython-311.pyc'
import '_sre' # <class '_frozen_importlib.BuiltinImporter'>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\__pycache__\_parser.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\_parser.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\re\\__pycache__\\_parser.cpython-311.pyc'
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\__pycache__\_constants.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\_constants.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\re\\__pycache__\\_constants.cpython-311.pyc'
import 're._constants' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE501950>
import 're._parser' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4F6990>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\__pycache__\_casefix.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\re\_casefix.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\re\\__pycache__\\_casefix.cpython-311.pyc'
import 're._casefix' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE50D090>
import 're._compiler' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4F3310>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\copyreg.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\copyreg.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\copyreg.cpython-311.pyc'
import 'copyreg' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE50E490>
import 're' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4F1590>
# C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\__pycache__\token.cpython-311.pyc matches C:\Users\rico\AppData\Local\Programs\Python\Python311\Lib\token.py
# code object from 'C:\\Users\\rico\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\__pycache__\\token.cpython-311.pyc'
import 'token' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4F2B90>
import 'tokenize' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE4DE590>
import 'linecache' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE26E0D0>
import 'inspect' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE1CEB10>
import 'rlcompleter' # <_frozen_importlib_external.SourceFileLoader object at 0x00000279FE1C2AD0>
Python 3.11.1 (tags/v3.11.1:a7a450f, Dec  6 2022, 19:58:39) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

# Your environment

- CPython versions tested on: 3.11.1
- Operating system and architecture: Windows 11 Pro 22H2 11th Gen Intel(R) Core(TM) i7-11700F 64-bit operating system, x86-64-based processor

Be sure to review the [contributing guidelines](https://github.com/python/cpython/blob/d9dff4c8b5ab41c47af002ad7fb083c953e75f31/.github/CONTRIBUTING.rst) and [code of conduct](https://github.com/python/.github/blob/d043932e7db760a58b6005436e2eb969077631f5/CODE_OF_CONDUCT.md).

no

**Comment by ricolay** [comment]
holy sh9t why did it format like that

**Comment by sobolevn** [comment]
Sorry, not clear what the bug is. Can you please clarify?

**Comment by ricolay** [comment]
This is what I see in console when I type the commands “python” and “py -v”

It returns a huge error I (assume)
When I do `python -version` it thinks I am doing `python -e`

**Comment by sobolevn** [comment]
Aren't you confusing `-v` (verbose) and `-V` (same as `--version`)?

From `python --help`:

```
-v     : verbose (trace import statements); also PYTHONVERBOSE=x
         can be supplied multiple times to increase verbosity
-V     : print the Python version number and exit (also --version)
```

**Comment by ricolay** [comment]
Ah. I had wondered if it was just verbose logging, but I had an issue
previously so was worried. Let me try again. Whoops sorry for wasting time

Solved, close

**Comment by sobolevn** [comment]
No worries, happy to help :)
